diesel/src/result.rs

```diff
@@ -45,6 +45,8 @@ pub enum Error {
     /// An example of when this error could occur is if you are attempting to
     /// construct an update statement with no changes (e.g. all fields on the
     /// struct are `None`).
+    ///
+    /// When using `optional_empty_changeset`, this error is turned into `None`.
```

Comment:
Minor note: That comment is not correct anymore. It might be a got idea to move this comment to the new error type.

Reply:
K, will do.